### PR TITLE
Olena Zhovtiuk week1 homework ecommerce

### PR DESCRIPTION
**Pull request by OlenaZH84**
Good evening.
My first ecommerce homwork.

### CODE REVIEW

File: week1/project/ecommerce/src/components/Categories.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import categories from "../fake-data/all-categories";
+ 
+export default function Categories({onClickCategory, activeCategory}) {
+
+ const setCategoryName = (name) => name.slice(6);
+  function setSelected(items, activeCategory){
+      console.log(items);
+       if (activeCategory === 'all' || items!=activeCategory){
```
Comment (ObadaMoustafa) [inline week1/project/ecommerce/src/components/Categories.js]: something wrong in the statement `!=` should be `!==`
✏️need to modify✏️

File: week1/project/ecommerce/src/components/Categories.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      }
+  } 
+  
+ return (
+    <div className='categories'>
+      
+      {categories.map((items, index)=> {
+      const categoryName=setCategoryName(items);
+      const selectedNotSelected=setSelected(categoryName, activeCategory);
+       return  <div className={selectedNotSelected}   key={`${categoryName}_${index}`} onClick ={()=>onClickCategory(categoryName)} >{items}</div>
```
Comment (ObadaMoustafa) [inline week1/project/ecommerce/src/components/Categories.js]: is would be better if separated in another component
📝advise📝

File: week1/project/ecommerce/src/components/Products.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    }
+     
+ return (
+    <>
+    {
+       prodList.map((obj, index)=>{
+        return (
+        <li  key={index} className='products--item'>
+        <div className='product'>
+            <img className='product--image' src={obj.image}/>
```
Comment (ObadaMoustafa) [inline week1/project/ecommerce/src/components/Products.js]: any img element should has `alt` attribute
✏️need to modify✏️

File: week1/project/ecommerce/src/components/Products.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+            <span className='product--title'>{obj.title}</span>
+        </div>
+    </li>
+        )
+       })
```
Comment (ObadaMoustafa) [inline week1/project/ecommerce/src/components/Products.js]: also would be great if separated in another component
📝advise📝

**Review by ObadaMoustafa — COMMENTED** [review]
great work Olena just a small thing you need to modify but generally you did great

File: week1/project/ecommerce/src/components/Products.js (hunk 3 above)
Comment (OlenaZH84) [reply week1/project/ecommerce/src/components/Products.js]: fixed

File: week1/project/ecommerce/src/components/Categories.js (hunk 1 above)
Comment (OlenaZH84) [reply week1/project/ecommerce/src/components/Categories.js]: fixed

**Pull request closed without merging**